server/pulp/server/managers/consumer/agent.py

```diff
@@ -14,10 +14,13 @@
 """
 
 import sys
+
 from logging import getLogger
 from uuid import uuid4
+from gettext import gettext as _
```

Comment:
alphabetical?  I'm not aware of this requirement in pep8.  can you point me to the text?